Answers/hw03_stubs_do_not_edit.py

```diff
@@ -0,0 +1,85 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Megan Lucas
+# Username:Megan_Lucas
+#
+# Assignment:HW03_stub
+# Purpose: To build a bouquet of flowers with a bow
+# Google Doc Link:https://docs.google.com/document/d/1ZtouqxWRp3KCr7y-horbUReL1Hd0j0SWmor7oxk6oxs/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor("lightblue")
+m = turtle.Turtle()
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor("lightblue")
+m = turtle.Turtle()
+m.pendown
+### first stem
+m.speed(0)
+m.color("green")
+m.left(90)
+m.forward(100)
+m.right(180)
+m.forward(50)
+### second stem
+m.right(-45)
+m.forward(-20)
+m.right(180)
+m.forward(50)
+### third stem
+m.right(180)
+m.forward(70)
+m.left(90)
+m.forward(70)
+
+
+def draw_petal(petal1):
```

Comment:
I notice you have an input parameter called `petal1` which is probably supposed to be the turtle you are using to draw. But, you use the turtle `m` to draw (which was defined at global scope, and will not be accessible once you move it into main). 

Convert all of the `m` turtles inside this function to `petal1`. Below, I have a comment for fixing the scoping issue for `m` inside `main` becoming `petal1` in here.